ui/src/components/photoGallery/ProtectedMedia.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+const isNativeLazyLoadSupported = false //'loading' in HTMLImageElement.prototype
```

Comment:
I don't understand the purpose of this variable. Does it ever change if it's const?

Reply:
This variable is used to check if native lazy load support via <img loadin="lazy" /> attribute supported.
For testing purposes, it was hardcoded to false in order to check the fallback implementation of IntersectionObserver.
Fixed it.